.github/workflows/windows.yml

```diff
@@ -31,6 +31,13 @@ jobs:
     runs-on: windows-2022
     name: Download testing bundle
     steps:
+      - name: Set commit status as pending
+        uses: myrotvorets/set-commit-status-action@v2.0.0
+        with:
+          sha: ${{ github.sha }}
```

Comment:
Although this is not guaranteed to match the commit the testing bundle was built with...